algorithms/src/Kokkos_Sort.hpp

```diff
@@ -53,6 +53,22 @@
 
 #include <algorithm>
 
+#if defined(KOKKOS_ENABLE_CUDA)
+#include "std_algorithms/Kokkos_BeginEnd.hpp"
+
+// Workaround for `Instruction 'shfl' without '.sync' is not supported on
+// .target sm_70 and higher from PTX ISA version 6.4`.
+// Also see https://github.com/NVIDIA/cub/pull/170.
+#if ((__CUDACC_VER_MAJOR__ >= 9) || defined(__NVCOMPILER_CUDA__) || \
+     CUDA_VERSION >= 9000) &&                                       \
```

Comment:
`CUDA_VERSION` will always be larger than `9000`. The (new) minimum is `Cuda 10.0.0`.

Reply:
Removed.